spec/support/shared/multiple_model_examples.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples 'multiple models' do
+  let(:enum_name) { :status }
+  let(:plural_enum_name) { enum_name.to_s.pluralize }
+  let(:first_model_values) { { active: 'First Active', inactive: 'First Inactive' } }
+  let(:second_model_values) { { pending: 'Second Pending', completed: 'Second Completed' } }
+
```

Comment:
suggestion: please use a similar setup as `spec/support/shared/translation_examples.rb`, for consistency.